LICENSE

```diff
@@ -207,6 +207,6 @@ Licenses for dependency projects can be found here:
 
 ---------------
 
-akka-protobuf contains the sources of Google protobuf 2.5.0 runtime support,
+protobuf contains the sources of Google protobuf 2.5.0 runtime support,
```

Comment:
Could this be changed to`pekko-protobuf module contains`?  The dir rename makes this statement very ambiguous. The Notice should be use the jar name as opposed to the dir name.

Reply:
Done